R/dplyr.R

```diff
@@ -68,9 +68,11 @@ mutate.dm <- function(.data, ...) {
 #' @export
 mutate.zoomed_dm <- function(.data, ...) {
   tbl <- get_zoomed_tbl(.data)
-  mutated_tbl <- mutate(tbl, ...)
+  quos <- enquos(...)
```

Comment:
Same as in `mutate.tbl_df()`:

```suggestion
  quos <- enquos(..., .named = TRUE)
```